README.md

```diff
@@ -12,6 +12,9 @@
 Создаем новый файл *local_config.py*, в него копируем все из *example_config.py*, меняем
 *local_config.py* 
 
+### Запуск
+Для запуска проекта выполните команду `python3 -m money_bot`
```

Comment:
When you're using the virtualenv's python, the python is can be accessed as just `python`